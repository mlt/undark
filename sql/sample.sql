create table sample (
  pkey integer primary key,
  c1 real,
  c2 integer,
  c3 text,
  c4 blob,
  c5 numeric
);

insert into sample values
(-42, 1234.567, 12, 'hello', 'blob', '123.45'),
(1234567, 1234.567, 12, 'hello', 'blob', '123.45'),
(-1234567, -1234.567, 0xcccc, 'world', 'blob2', '12345'),
(1234567890, 123456.7, 0xcccccc, 'foo', 'bar', '12345e-4'),
(-1234567890, '123456.7zx', 0xcccccccc, 'baz', 'bar', '12345e-4cv'),
(12345678901, '123456zx', 0xfcccccccc, 'baz', 'bar', '12345e-4cv');

PRAGMA auto_vacuum = NONE;
PRAGMA secure_delete=OFF;

delete from sample;
