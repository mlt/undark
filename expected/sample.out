12345678901,NULL,"123456zx",-858993460,"baz","bar","12345e-4cv"
-1234567890,NULL,"123456.7zx",3435973836,"baz","bar","12345e-4cv"
1234567890,NULL,123456.700000,13421772,"foo","bar",1.234500
-1234567,NULL,-1234.567000,52428,"world","blob2",12345
1234567,NULL,1234.567000,x12,"hello","blob",123.450000
-42,NULL,1234.567000,x12,"hello","blob",123.450000
